packages/dashboard/src/components/FileUpload/index.tsx

```diff
@@ -2,6 +2,7 @@ import { MaskColorVar } from '@masknet/theme'
 import { makeStyles, Typography } from '@material-ui/core'
 import { useEffect, useState } from 'react'
 import { File as FileIcon } from '@masknet/icons'
+import { toText } from '@dimensiondev/kit'
```

Comment:
The function name `toText` is not clear enough, what's the source type? Add a namespace or function name like `blobToText` would be better.